mass_mailing_switzerland/models/recurring_contract.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Nathan Fluckiger <nathan.fluckiger@hotmail.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields, api
+
+
+class RecurringContract(models.Model):
+
+    _inherit = 'recurring.contract'
+
+    campaign_id = fields.Many2one('utm.campaign')
```

Comment:
This is not needed because `recurring.contract` inherits from `utm.mixin` and this adds all relation to utm objects.